tests/NoneTest.elm

```diff
@@ -0,0 +1,16 @@
+{--
```

Comment:
i think this file needs to be removed, i think its from the original branch but has since been removed

Reply:
Removing.